seed/utils/merge.py

```diff
@@ -0,0 +1,261 @@
+# !/usr/bin/env python
```

Comment:
Much of the logic here existed previously. These changes were made to isolate it for reuse. Testing was set up before this refactor to help ensure functionality remained the same.